test/test_prototype_transforms_functional.py

```diff
@@ -11,19 +11,31 @@
 make_tensor = functools.partial(torch.testing.make_tensor, device="cpu")
 
 
-def make_image(size=None, *, color_space, extra_dims=(), dtype=torch.float32):
+def make_image(size=None, *, color_space, extra_dims=(), dtype=torch.float32, constant_alpha=True):
     size = size or torch.randint(16, 33, (2,)).tolist()
 
-    num_channels = {
-        features.ColorSpace.GRAYSCALE: 1,
-        features.ColorSpace.RGB: 3,
-    }[color_space]
+    try:
+        num_channels = {
+            features.ColorSpace.GRAYSCALE: 1,
+            features.ColorSpace.GRAYSCALE_ALPHA: 1,
+            features.ColorSpace.RGB: 3,
+            features.ColorSpace.RGBA: 3,
+        }[color_space]
```

Comment:
Reading this first time, I found it hard to understand why the alpha values are the way they are. We should refactor on a follow up PR to make this more obvious. One approach could be to generate the right number of dimentions for each channel (gray-alpha=2, rbga=4) but then overwrite the final channel with white. That approach will be easier to understand than initializing an RGBA with 3 channels and concatenating an alpha.